projects/pages/src/public-api.ts

```diff
@@ -0,0 +1,9 @@
+export * from './modules/action-hub/action-hub.module';
+export {SkyActionHubComponent as λ1} from './modules/action-hub/action-hub.component';
+export {SkyActionHubButtonsComponent as λ2} from './modules/action-hub/action-hub-buttons.component';
+export {SkyActionHubContentComponent as λ3} from './modules/action-hub/action-hub-content.component';
+export * from './modules/action-hub/types/action-hub-needs-attention';
+export * from './modules/action-hub/types/page-link';
+export * from './modules/action-hub/types/recent-link';
+export * from './modules/page-header/page-header.module';
+export {SkyPageHeaderComponent as λ4} from './modules/page-header/page-header.component';
```

Comment:
The "lambda" exports should include a comment, placed at the bottom of the file, see: https://github.com/blackbaud/skyux-core/blob/5.0.0-next/projects/core/src/public-api.ts#L70-L74